### PR TITLE
Added Basic contextMenus sample

### PR DESCRIPTION
**Pull request by IanStanion-google**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Can you please run `npm run lint:fix` to unblock the build? You might have to do a few manual fixes as well in case eslint can fix everything automatically.

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thanks for updating this! Left some initial thoughts. Could we add a README too (see api-samples/scripting for an example)? 😁

**Comment by jpmedley** [comment]
@oliverdunk, Do have time to include a README using the new template? https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/README-template.md

File: api-samples/contextMenus/basic/sample.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  type: "checkbox",
+  id: "checkbox",
+});
+
+// Intentionally create an invalid item, to show off error checking in the
+// create callback.
+chrome.contextMenus.create(
+  { title: "Oops", parentId: 999, id: "errorItem" },
+  function () {
+    if (chrome.extension.lastError) {
```
Comment (oliverdunk) [inline api-samples/contextMenus/basic/sample.js:89]: Can we use `chrome.runtime.lastError`? `chrome.extension` was just an alias that has been deprecated for a while :)

```suggestion
    if (chrome.runtime.lastError) {
```

File: api-samples/contextMenus/basic/sample.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+// Create a checkbox item.
+var checkbox1 = chrome.contextMenus.create({
+  title: "checkbox",
+  type: "checkbox",
+  id: "checkbox",
+});
+
+// Intentionally create an invalid item, to show off error checking in the
+// create callback.
```
Comment (oliverdunk) [inline api-samples/contextMenus/basic/sample.js:85]: Does this error check work for you? On my side it doesn't seem to.

**Comment by IanStanion-google** [comment]
@jpmedley Added readme file based off your template.

File: api-samples/contextMenus/basic/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# chrome.contextMenus
+
+A sample extension demonstrating use the chrome.contextMenu API.
```
Comment (oliverdunk) [inline api-samples/contextMenus/basic/README.md:3]:
```suggestion
A sample extension demonstrating use of the chrome.contextMenus API.
```

**Review by oliverdunk — APPROVED** [review]
Left one nit on the README, otherwise this looks perfect! Thanks so much for going through a few rounds of reviews on this :)

**Pull request merged**